Source/WebInspectorUI/UserInterface/Views/DetailsSection.js

```diff
@@ -73,11 +74,17 @@ WI.DetailsSection = class DetailsSection extends WI.Object
 
     get title()
     {
-        return this._titleElement.textContent;
+        return this._titleElement?.textContent;
     }
 
     set title(title)
     {
+        if (!this._titleElement) {
+            this._titleElement = document.createElement("span");
+            this._headerElement.appendChild(this._titleElement);
```

Comment:
we might want to always create the `_titleElement` so that we don't have to worry about it being inserted into the wrong spot (i.e. out of order) if the `title` is set later on

Reply:
Reverted back to the original where `_titleElement` is created anyway. We now toggle the `hidden` class for `_headerElement` whenever the title is set. If falsy, add `.hidden`, otherwise, remove `.hidden`.